catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java

```diff
@@ -123,6 +128,22 @@ public boolean onOptionsItemSelected(MenuItem item) {
 		return super.onOptionsItemSelected(item);
 	}
 
+	public boolean isRecoveredFromCrash() {
+		return PreferenceManager.getDefaultSharedPreferences(this).getBoolean(RECOVERED_FROM_CRASH, false);
+	}
+
+	public boolean checkIfCrashRecoveryAndFinishActivity(final Activity context) {
```

Comment:
Please make private, change return type to void and remove the parameter.

Previously I was thinking about passing in the context (or SharedPreferences) into `isRecoveredFromCrash`. However, having this duplicate code (`PreferenceManager.getDefaultSharedPreferences(this)`) doesn't look bad after all. So, no need to pass in the context here nor pass any parameter to `isRecoveredFromCrash`.